src/content/docs/en/reference/programmatic-reference.mdx

```diff
@@ -246,4 +277,59 @@ import { sync } from "astro";
 await sync({
   root: "./my-project",
 });
-```
\ No newline at end of file
+```
+
+## `mergeConfig()`
+
+<p>
+
+**Type:** `<T extends AstroConfig | AstroInlineConfig>(config: T, overrides: DeepPartial<T>) => T`
+</p>
+
+Imported from `astro/config`, merges a partial Astro configuration on top of an existing, valid, Astro configuration.
+
+- General options are overriden.
```

Comment:
```suggestion
- General options in the existing config are overriden.
```
(Also, I'm not sure what "general" options are? Is that a specific thing? If not, is there a proper name for these options? "top-level"?)

Reply:
Anything that doesn't match one of the other rules. Even if they are nested.

Reply:
OK, then should this point come *last*? As in:

- All other options override the existing config